lib/open_food_network/reports/line_items.rb

```diff
@@ -23,10 +23,9 @@ def list(line_item_includes = nil)
         end
 
         editable_line_items = editable_line_items(line_items)
+        without_editable_line_items = line_items.reject { |li| editable_line_items.include? li }
```

Comment:
we are returning the line items that are not in `editable_line_items`, right? I read this logic so many times that now I'm not sure about anything :joy:

Reply:
Well, shouldn't this work?
```suggestion
        without_editable_line_items = line_items - editable_line_items
```

Reply:
Don't ask me why, but it didn't. I'll give it another try because it's possible that the other things I changed when I tried that were conflicting with it.

Reply:
it did!